numpy/__init__.py

```diff
@@ -202,9 +202,8 @@ def __getattr__(attr):
                 from .testing import Tester
                 return Tester
             else:
-                raise AttributeError(
-                    "module %s has no attribute $s".format(__name__, attr))
-
+                raise AttributeError("module {0!r} has no attribute "
```

Comment:
Using `0` and `1` placeholders is something I rarely see - I'd be tempted to omit them

Reply:
Not sure why I used them here.  I'll take them out.